### PR TITLE
Update 01-login.md

### PR DESCRIPTION
**Pull request by joshcanhelp**
Replicating change from the sample:

https://github.com/auth0-samples/auth0-laravel-php-web-app/pull/36

### CODE REVIEW

File: articles/quickstart/webapp/laravel/01-login.md [hunk 1]
```diff
@@ -327,23 +327,11 @@ class CustomUserRepository extends Auth0UserRepository
...
-            $user->setAttribute( 'email', $profile['email'] );
-            $user->setAttribute( 'sub', $profile['sub'] );
-            $user->setAttribute( 'name', isset( $profile['name'] ) ? $profile['name'] : '' );
-
-            $user->save();
-        }
-        return $user;
+    protected function upsertUser( $profile ) {  
+        return User::firstOrCreate(['sub' => $profile['sub']], [
+            'email' => $profile['email'] || '',
```
Comment (stevehobbsdev) [inline articles/quickstart/webapp/laravel/01-login.md]: The sample this is replicated from doesn't default `email` to an empty string - is that significant and should the code blocks be identical?

Comment (joshcanhelp) [reply articles/quickstart/webapp/laravel/01-login.md]: You know what ... they're both wrong 😆 

https://github.com/auth0-samples/auth0-laravel-php-web-app/pull/38

Comment (joshcanhelp) [inline articles/quickstart/webapp/laravel/01-login.md]:
```suggestion
            'email' => $profile['email'] ?? '',
```

File: articles/quickstart/webapp/laravel/01-login.md [hunk 2]
```diff
@@ -327,23 +327,11 @@ class CustomUserRepository extends Auth0UserRepository
...
+            'name' => $profile['name'] || '',
```
Comment (joshcanhelp) [inline articles/quickstart/webapp/laravel/01-login.md]:
```suggestion
            'name' => $profile['name'] ?? '',
```

**Review by stevehobbsdev — APPROVED** [review]

**Pull request merged**